code_schemes/constituency.json

```diff
@@ -0,0 +1,214 @@
+{
+    "SchemeID": "Scheme-4c5b955d",
+    "Name": "Constituency",
+    "Version": "0.0.0.1",
+    "Codes": [
+      {
+        "CodeID": "code-bd610414",
+        "CodeType": "Normal",
+        "DisplayText": "Mwea Constituency",
```

Comment:
Do you need to include "Constituency" all of the display texts?

(I'm not qualified to know the answer, just making sure there's a reason)

Reply:
Actually there is no need.